src/main/java/Task.java

```diff
@@ -0,0 +1,39 @@
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {   //ok to leave as public?
```

Comment:
Yes you need to leave it as public. But do remove such comments from your code as it goes against the java coding standard. 👍